common/app/routes/Challenges/views/backend/Back-End.jsx

```diff
@@ -136,7 +136,7 @@ export class BackEnd extends PureComponent {
               onSubmit={ handleSubmit(executeChallenge) }
               >
               <SolutionInput
-                placeholder='https://your-app.com'
+                placeholder='https://your-app.glitch.me'
```

Comment:
@BerkeleyTrue @QuincyLarson @Bouncey do we have any hard requirements to use glitch URLs only? I am thinking there should be flexibility to allow for others (heroku, etc.)

Reply:
Not a priority though.

Reply:
The only requirement is that the camper uses the boilerplate, as this includes a route for our tests to access the source files.